client/src/components/Banner.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+
+class Banner extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      photos: [{url: ''}],
+      galleryOpen: false
```

Comment:
I believe trailing commas preferred after last item in state declaration (and after last item in all instances of setState below)